File: app/src/main/java/com/example/bottlerocketstudioscodingtest/UI/DetailActivity.java
package com.example.bottlerocketstudioscodingtest.UI;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.example.bottlerocketstudioscodingtest.R;

public class DetailActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setTitle(R.string.detail_activity_label);
        setContentView(R.layout.activity_detail);

        if (savedInstanceState == null) {
            DetailFragment storeDetails = new DetailFragment();

            Bundle bundle = getIntent().getExtras();
            storeDetails.setArguments(bundle);

            getSupportFragmentManager().beginTransaction()
                    .add(R.id.detail_fragment, storeDetails)
                    .commit();
        }
    }
}

File: app/src/main/java/com/example/bottlerocketstudioscodingtestkotlin/UI/DetailActivity.kt
package com.example.bottlerocketstudioscodingtestkotlin.UI

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

import com.example.bottlerocketstudioscodingtestkotlin.R

class DetailActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setTitle(R.string.detail_activity_label)
        setContentView(R.layout.activity_detail)

        if (savedInstanceState == null) {
            val storeDetails = DetailFragment()

            val bundle = intent.extras
            storeDetails.arguments = bundle

            supportFragmentManager.beginTransaction()
                    .add(R.id.detail_fragment, storeDetails)
                    .commit()
        }
    }
}
